binder/environment.yml

```diff
@@ -1,26 +1,201 @@
 name: TDS
 channels:
+  - conda-forge
   - defaults
 dependencies:
-  - python=3.7
-  - requests
-  - numpy
-  - pandas
-  - geopandas
-  - descartes
-  - scipy
-  - matplotlib
-  - seaborn
-  - openpyxl
-  - jupyter
-  - autopep8
-  - conda-forge::jupyter_contrib_nbextensions
-  - pymc3
-  - scikit-learn
+  - brotli=1.0.9
```

Comment:
In order to get the conda environment to work on an m1 mac I had to make the following changes:
```diff
diff --git a/binder/environment.yml b/binder/environment.yml
index 726920e..7705069 100644
--- a/binder/environment.yml
+++ b/binder/environment.yml
@@ -51 +50,0 @@ dependencies:
-  - python=3.9.13
@@ -53 +51,0 @@ dependencies:
-  - python_abi=3.9
@@ -95 +93 @@ dependencies:
-    - fiona==1.8.21
+    - fiona>=1.8.21
@@ -125 +123 @@ dependencies:
-    - matplotlib==3.5.2
+    - matplotlib>=3.5.2
@@ -157 +155 @@ dependencies:
-    - pyproj==3.3.1
+    - pyproj>=3.3.1
@@ -164,2 +162,2 @@ dependencies:
-    - pyzmq==23.2.0
-    - rasterio==1.3.2
+    - pyzmq>=23.2.0
+    - rasterio>=1.3.2
@@ -169 +167 @@ dependencies:
-    - ruamel-yaml-clib==0.2.6
+    - ruamel-yaml-clib==0.2.7
```